docs-rtd/howto/manage-roles.md

```diff
@@ -1,5 +1,5 @@
-(manage-groups)=
-# Manage groups
+(manage-roles)=
+# Manage roles
 
 ```{note}
 In the Juju ecosystem, roles are supported only when using [JAAS](https://canonical-jaas-documentation.readthedocs-hosted.com/en/latest/).
```

Comment:
There are two more issues:

In https://github.com/juju/terraform-provider-juju/commit/876a33492d8729f92ef1e71b639510352a3d9a23#diff-3e73c1055d11e6b4b7d6da83aea57c7842f52c6c322de1e169c46411d747d7bdR60 remove group from the title and the group link from the body.

In https://github.com/juju/terraform-provider-juju/commit/876a33492d8729f92ef1e71b639510352a3d9a23#diff-ebfba5ccfbf0b48c73ab0a79752334db3551b6a133760959a6f0f73e1052e4f9R36 remove mention of roles.